src/main/java/org/jabref/gui/collab/EntryChangeViewModel.java

```diff
@@ -21,15 +27,14 @@ class EntryChangeViewModel extends DatabaseChangeViewModel {
     private final BibEntry newEntry;
     private ThreeWayMergeView threeWayMergeView;
 
+    private DialogService dialogService;
+
     public EntryChangeViewModel(BibEntry entry, BibEntry newEntry) {
-        super();
+        super(entry.getCitationKey().map(key -> Localization.lang("Modified entry") + ": '" + key + '\'')
```

Comment:
Please add this key as a parameter to the variable. Other languages might have a different word order/sentence structure...